core/new-gui/src/app/dashboard/type/project.ts

```diff
@@ -0,0 +1,7 @@
+export interface Project
+  extends Readonly<{
+    pid: number;
+    name: string;
+    ownerId: number;
```

Comment:
please use `ID` instead of `Id` in all naming conventions.

Reply:
done